packet.go

```diff
@@ -917,6 +917,26 @@ func (p *StatVFS) MarshalBinary() ([]byte, error) {
 	return buf.Bytes(), err
 }
 
+type sshFxpFsyncPacket struct {
+	ID     uint32
+	Handle string
+}
+
+func (p sshFxpFsyncPacket) id() uint32 { return p.ID }
+
+func (p sshFxpFsyncPacket) MarshalBinary() ([]byte, error) {
```

Comment:
The marshalling of this packet is performed differently from those for `sshFxpExtendedPacketStatVFS`, `sshFxpExtendedPacketPosixRename`, and `sshFxpExtendedPacketHardlink`.

I would recommend revising it to work the same way as the other extended packets.

Reply:
I'm not sure I understand. I looked at sshFxpStatvfsPacket.MarshalBinary and adapted that. Aren't the sshFxpExtendedPacket* for server use? It looks like they have UnmarshalBinary but not MarshalBinary.

Reply:
🤦‍♀️